skimage/feature/tests/test_texture.py

```diff
@@ -138,21 +138,21 @@ def test_contrast(self):
                               normed=True, symmetric=True)
         result = np.round(result, 3)
         contrast = greycoprops(result, 'contrast')
-        np.testing.assert_almost_equal(contrast[0, 0], 0.586)
+        np.testing.assert_almost_equal(round(contrast[0, 0], 3), 0.585)
```

Comment:
`assert_almost_equal` should have a `digits` keyword, so that you don't have to use `round`